.github/workflows/build.yml

```diff
@@ -154,12 +154,13 @@ jobs:
         sleep 3
         herbstluftwm &
         sleep 1
-        python -c "import sys; print(sys.path)"
+        rm -rf res
```

Comment:
The commit message of this change is a bit vague about why this is needed, and perhaps a bit misleading about what is done as well.

Reply:
This seems to be necessary because in certain situations tests would look inside this directory for .so files, and not find them. Why they were doing that, i do not know, and it would only happen some of the time, and only when running in parallel. Probably, some tests were running in unexpected folders due to insane chdir-ing all over the place, which would arbitrarily work with the normal running order, but would not always work when running in parallel, as then the order will be different.

Reply:
A small comment about the rm would be nice then.